packages/venia-concept/webpack.config.js

```diff
@@ -56,7 +63,8 @@ module.exports = async env => {
             STORE_NAME: JSON.stringify('Venia'),
             STORE_VIEW_CODE: process.env.STORE_VIEW_CODE
                 ? JSON.stringify(process.env.STORE_VIEW_CODE)
-                : JSON.stringify(storeConfigData.code)
+                : JSON.stringify(storeConfigData.code),
+            AVAILABLE_STORE_VIEWS: JSON.stringify(availableStores)
```

Comment:
To ensure storybook continues to function, changes to the webpack config such as global props or new queries must also be applied to the [storybook webpack config](https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/.storybook/webpack.config.js)

Reply:
Updated :)